src/main/java/com/synopsys/integration/alert/web/actions/SystemActions.java

```diff
@@ -74,6 +85,40 @@ public SystemActions(final SystemStatusUtility systemStatusUtility, final System
 
     private SystemMessageModel convert(final SystemMessage systemMessage) {
         final String createdAt = RestConstants.formatDate(systemMessage.getCreated());
-        return new SystemMessageModel(systemMessage.getType(), createdAt, systemMessage.getContent());
+        return new SystemMessageModel(systemMessage.getSeverity(), createdAt, systemMessage.getContent(), systemMessage.getType());
+    }
+
+    public SystemSetupModel getCurrentSystemSetup() {
+        final RequiredSystemConfiguration systemConfiguration = systemInitializer.getCurrentSystemSetup();
+        return new SystemSetupModel(systemConfiguration.getBlackDuckProviderUrl(),
+            systemConfiguration.getBlackDuckConnectionTimeout(),
+            systemConfiguration.getBlackDuckApiToken(),
+            StringUtils.isNotBlank(systemConfiguration.getBlackDuckApiToken()),
+            systemConfiguration.getGlobalEncryptionPassword(),
```

Comment:
I don't think we want to return the BlackDuckApiToken, GlobalEncryptionPassword, GlobalEncryptionSalt, or the ProxyPassword. We only want to return if they are set since this is an unauthenticated endpoint that will be getting this information.

Reply:
We never want to return any kind of sensitive information to the UI authenticated or not.